contributing/contributing-code.md

```diff
@@ -122,3 +134,43 @@ We recommend you handle signing/being covered by a CLA *before* you send a PR to
   * If you aren't sure whehther your company has already signed a corporate CLA, if you don't know who at your company owns the Google Group you need to be added to, or you run into any other questions about the corporate CLA, please reach out to @mrjoro (via cc on a PR or on [Slack](https://amphtml.slack.com) ([signup](https://bit.ly/amp-slack-signup))) or the [#contributing](https://amphtml.slack.com/messages/C9HRJ1GPN/details/) channel on Slack.
+
+## Code review and approval
+
+All code in AMP must be reviewed and approved before it is merged so that we can maintain AMP's high quality and consistency.
+
+This section describes the different review/approval roles and the approvals that are required before code may be merged.  These guidelines are specific to the amphtml repository.  Other ampproject repos may follow the same guidelines or use different guidelines as documented in their CONTRIBUTING.md files.
+
+### Roles
+
+* Owners
+  * Review & approve PRs in the area in which they have expertise.
+  * Requirements to be an Owner:
+    * Demonstrated expertise in the area in which they are an OWNER.
+    * Any GitHub user (including those who are not Reviewers or Collaborators) may be an Owner.
+    * When creating a new directory (such as when creating a new AMP extension) the author of the pull request should designate themselves as an Owner of that directory.
+    * Owners of an area may approve other owners at or below their area of expertise following the normal PR process.
+  * The list of Owners for a directory can be found in the [OWNERS.yaml](https://github.com/ampproject/amphtml/search?o=asc&q=filename%3AOWNERS.yaml&s=indexed) file in the directory or a parent directory.
+
+* Reviewers
+  * Review, approve and merge PRs in the repo for which Reviewer status is granted.
+  * Reviewer status is granted to folks who have demonstrated deep familiarity with the code-style and conventions of the respective repository.
+  * A person may become a Reviewer after 10 merged PRs or 10 high quality reviews of complex PRs and a +1 from 1 current REVIEWER.  Qualifying PRs must be non-trivial (not only fixing typos, not only config changes) and should have implemented or documented at least 2 new features.
+  * The list of Reviewers is maintained in the [Reviewers (amphtml)](https://github.com/orgs/ampproject/teams/reviewers-amphtml) GitHub team.
+
+* Collaborators
+  * Review, approve and merge PRs in the repo for which COLLABORATOR status is granted.
+  * Collaborator status is granted to folks who have proven basic familiarity with the respective repository.
+  * A person may become a Collaborator after 2 merged PRs that are non-trivial (not only fixing typos, not only config changes) and a +1 from 1 current Reviewer.
+  * The list of Collaborators is maintained in the [Collaborators (amphtml)](https://github.com/orgs/ampproject/teams/collaborators-amphtml) GitHub team.
```

Comment:
We need to document how to apply!

Reply:
Good point... how do we want to handle that?  File an issue in the given repo (e.g. amphtml) and cc ampproject/collaborators-amphtml ?

Reply:
Sorry, missed this comment. I think filing an issue is good but CCing the full list is probably too much. Probably better to CC reviewers.

Reply:
I updated each of the roles to document how to request access.  PTAL.